db-migration/src/lib.rs

```diff
@@ -70,7 +69,7 @@ impl Migrations {
                     let mpbc = Arc::clone(&mpb);
                     let pb = move |count: u64| -> ProgressBar {
                         let pb = mpbc.add(ProgressBar::new(count));
-                        pb.set_draw_target(ProgressDrawTarget::to_term(Term::stdout(), None));
+                        pb.set_draw_target(ProgressDrawTarget::stdout());
```

Comment:
I'm not sure is this ok.

Reply:
cc @zhangsoledad

Reply:
I tested it, the user experience is average, and I feel that the buffer effect is not good, I changed it back